exaudfclient/base/script_options_parser/ctpg/script_option_lines_ctpg.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SCRIPTOPTIONLINESCTPG_H
+#define SCRIPTOPTIONLINESCTPG_H
+
+#include <string>
+#include <functional>
+#include <vector>
+#include <map>
+#include <ostream>
+
+namespace ExecutionGraph
+{
+
+namespace OptionsLineParser
+{
+
+namespace CTPG
+{
+
+class ParserResult;
+
+struct ScriptOption {
+    std::string value;
+    size_t idx_in_source;
+    size_t size;
+
+    bool operator==(const ScriptOption & right) const {
+        return value == right.value && idx_in_source == right.idx_in_source && size == right.size;
+    }
+    friend void PrintTo(const ScriptOption& option, std::ostream* os) {
+        *os << "(" << option.value << "," << option.idx_in_source << "," << option.size << ")";
+    }
+};
+
+using options_t = std::vector<ScriptOption>;
+
+using options_map_t = std::map<std::string, options_t>;
+
+/*!
+ * \brief parseOptions Extracts syntactically valid option lines of form %<option> <values> [<values>] from UDF scripts.
```

Comment:
was this basically only one value which can have spaces?

Reply:
True, I will fix the documentation

Reply:
Done